### PR TITLE
Add allowDisagreed to props of PSClickWrap

### PR DESCRIPTION
**Pull request by cdhcs1516**
### Description
Relate to IRON-10390.

We do not make it explicit inside our react sdk that the onInvalid event (linked to the sending of disagreed events) will only fire when the underlying clickwrap group has the allow_disagreed property enabled. This property is currently API only in classic, although it is exposed in the new experience. 

This PR does following actions:
- Log a warning to the browser console when a function handler is registered for onInvalid events if the `allow_disagreed` property is not set.
- Update our documentation in that repo to call out this behavior
- Update the group properties to allow `allow_disagreed` to be set client side via the sdk

Tested in demo site:
- When `onInvalid` is passed without setting `allowDisagreed`, a console warning will be shown.
- When `allowDisagreed` is set, `onInvalid` events can be handled properly. Also, `disagreed` activities will be shown in group's activity tab if the user unchecks the box.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ import { PSBrowseWrap } from '@pactsafe/pactsafe-react-sdk'
...
 |      `testMode`      | Enable this to register any contract acceptances as test data that can be cleared within the PactSafe UI                                                                                                                                                                                                                                                                           |                                     bool                                     |                      No                     |                     false                    |
+|      `allowDisagreed`      | Enable this to allow an invalid event is triggered when user uncheck the box                                                                         |                                     bool                                     |                   If `onInvalid` is passed                      |                     undefined                    |
```
Comment (timothymorse) [inline README.md:142]: Might want to reword this to something like: "Enable this to allow invalid events to be triggered when a signer unchecks a checkbox." or something along those lines.

File: tests/PSClickWrap.test.js [hunk 2]
```diff
@@ -180,6 +180,16 @@ describe('PSClickWrap _ps interface tests', () => {
...
+  it('sets allowDisagreed to true if passed as a prop', () => {
+    mount(<PSClickWrap accessId="0000000-000000-0000-0000000" groupKey="example-clickwrap" signerId="test@abc.com" allowDisagreed />);
+    expect(_ps.mock.calls[1][2].allow_disagreed).toBe(true);
+  });
+
+  it('sets allowDisagreed to undefined if not passed as a prop', () => {
+    mount(<PSClickWrap accessId="0000000-000000-0000-0000000" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[1][2].allow_disagreed).toBeUndefined();
+  });
 });
```
Comment (Wilsonl9) [inline tests/PSClickWrap.test.js:193]: Is it possible to have a test where we try to pass in an onInvalid function without setting `allowDisagreed` to `true`?

**Review by Wilsonl9 — APPROVED** [review]
LGTM!

**Review by timothymorse — APPROVED** [review]

**Comment by AndriyPavliv** [comment]
Hi @cdhcs1516 ! Is there any ETA of this prop? Thanks.

**Comment by cdhcs1516** [comment]
@AndriyPavliv Hey! We just published a new version shipping this feature. Check the latest version 2.6.0.
